### PR TITLE
adding files

### PR DESCRIPTION
**Pull request by folguera**

### CODE REVIEW

**Comment by folguera** [comment]
This PR is needed for https://github.com/cms-sw/cmssw/pull/28950

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# L1Trigger-DTTriggerPhase2
```
Comment (silviodonato) [inline README.md:1]: @smuzaffar @mrodozov 
The name of the repository is `cms-data/L1Trigger-DTTriggerPhase2`
while here we are moving the `README.md` to ` # L1Trigger-DTPhase2Trigger `. Do you think this discrepancy can create problems?

**Comment by silviodonato** [comment]
A part from the comment above, this PR request is ready to be merged with https://github.com/cms-sw/cmssw/pull/30344

File: README.md (hunk 1 above)
Comment (mrodozov) [reply README.md:1]: None of our scripts are reading the README.md themselves to my knowledge. In short no

**Comment by mrodozov** [comment]
+externals
new files.

**Comment by silviodonato** [comment]
merge

**Pull request merged**